tests/testthat/test_featsel_filters.R

```diff
@@ -37,9 +37,7 @@ test_that("filterFeatures", {
   # extra test for rf.min.depth filter (#1066)
   fv = suppressWarnings(generateFilterValuesData(task = multiclass.task, method = "rf.min.depth",
     more.args = list("rf.min.depth" = c(method = "vh", conservative = "low"))))
-
   expect_class(fv, classes = "FilterValues")
-  expect_numeric(fv$data$value, any.missing = FALSE, all.missing = FALSE, len = getTaskNFeats(multiclass.task))
```

Comment:
The particular test that that check applies to is only for the RF min depth filter. So it should be OK to remove the test.
I have styled the file.

Reply:
`rf.min.depth` is only a naming deprecation, its the same as `randomForestSRC_var.select`.

You do not remove the whole test though, only the last `expect_numeric()` check. Why only this one?

Reply:
Because that was the only one that failed! :-) I don't know why those tests were put there originally, so I thought better to disturb as little as possible.

Reply:
Failing tests should be a trigger to remove such :)

The test looks reasonable for me: "Check if `fv$data$value` is numeric and has the same length as the task, including no NAs. So if this fail I think it is at least worth to take a closer look why this happens.

Reply:
It fails because NA's are returned. Previously that was not allowed, but with this change, the min depth filter will return NA's for those values that are below the threshold. Therefore this test is no longer valid. I did explain this above.

Reply:
Thanks. I checked the issue which the test refers to and by removing that line it should still be valid.

Sorry for having to state things twice, I am mixing things up/forgetting things while switching between all the PRs :smile: